en/flight_modes/mission.md

```diff
@@ -54,11 +54,18 @@ At high level all vehicle types behave in the same way when MISSION mode is enga
    Enter the time to wait after landing before disarming the vehicle.
    :::
 
-Missions can be paused by switching out of mission mode to any other mode (such as [Hold mode](../flight_modes/hold.md) or [Position mode](../flight_modes/position_mc.md).
-When you switch back to mission mode the vehicle will continue the mission, heading from the _current vehicle position_ to the current active mission item (the same waypoint it as heading towards originally).
-Note that if you moved the vehicle while the mission was paused you will no longer be following the original track towards the waypoint.
+Missions can be paused by switching out of mission mode to any other mode (such as [Hold mode](../flight_modes/hold.md) or [Position mode](../flight_modes/position_mc.md)), and resumed by switching back to mission mode.
+If the vehicle does not have camera trigger items, it will head from its _current position_ towards the same waypoint as it as was heading towards originally.
+If the vehicle has camera trigger items it will head from its current position towards the last waypoint it travelled through before pausing, and then retrace its path at the same speed and with the same camera triggering behaviour.
+This ensures that in survey/camera missions the planned path is captured.

```

Comment:
```suggestion
Missions can be paused by switching out of mission mode to any other mode (such as [Hold mode](../flight_modes/hold.md) or [Position mode](../flight_modes/position_mc.md)), and resumed by switching back to mission mode.
If the vehicle was not capturing images when it was paused, on resuming it will head from its _current position_ towards the same waypoint as it as was heading towards originally.
If the vehicle was capturing images (has camera trigger items) it will instead head from its current position towards the last waypoint it travelled through (before pausing), and then retrace its path at the same speed and with the same camera triggering behaviour.
This ensures that in survey/camera missions the planned path is captured.

```